awx/playbooks/project_update.yml

```diff
@@ -153,6 +153,39 @@
         - update_insights
         - update_archive
 
+- hosts: localhost
+  gather_facts: false
+  connection: local
+  name: Validate project content integrity
+  tasks:
+    - block:
+        - name: verify project integrity using GPG signature
+          playbook_integrity:
+            project_path: "{{ project_path | quote }}"
+            validation_type: gpg
+            gpg_pubkey: "{{ gpg_pubkey }}"
+          register: gpg_result
+
+        - name: GPG result
+          debug:
+            var: gpg_result
```

Comment:
I find the value of these debug statements kind of questionable. If you're in the UI, you can click on the "ok:" line in standard out.

![Screenshot from 2022-08-30 12-52-31](https://user-images.githubusercontent.com/1385596/187494953-ee523199-ebbd-4dd7-a2e7-5f229fac581c.png)

![Screenshot from 2022-08-30 12-52-38](https://user-images.githubusercontent.com/1385596/187495010-a3f2dcdf-53c3-4b1e-9dda-bed0e1f6d014.png)

So you're taking the return data from event data in one event and putting it into standard out in the next event, yes. But when you do that in a debug task, it's not much better for human readability. If I were using this on the regular, I'd think my preference would be to just have the one task, because the "ok:" output is probably sufficient at a high-level view.

Even if you do keep these debug tasks, I'd hope that the data could be narrowed down to _just_ echo the status or the fingerprint or something. This would be proportional to the debug task where we echo the repository version (which I also don't love). I think the JSON data is clearly too much, and my personal preference would be to delete the debug tasks entirely.

Reply:
:+1: on deleting the debug tasks